packages/jaeger-ui/src/components/common/CopyIcon.tsx

```diff
@@ -64,7 +64,7 @@ export default class CopyIcon extends React.PureComponent<PropsType, StateType>
     return (
       <Tooltip
         arrowPointAtCenter
-        mouseLeaveDelay={0.1}
+        mouseLeaveDelay={0}
```

Comment:
why?

Reply:
Because of this-

https://github.com/jaegertracing/jaeger-ui/assets/81948346/4e515d4c-6247-4cc5-a8db-0230b610a96d

Reply:
@yurishkuro what do you think?